scanner/image/db/Dockerfile

```diff
@@ -16,34 +22,29 @@ ENV PATH="$PATH:/usr/pgsql-$PG_MAJOR/bin/" \
     PGDATA="/var/lib/postgresql/data/pgdata"
 ENV LANG=en_US.utf8
 
+COPY signatures/RPM-GPG-KEY-PGDG-13 /
 COPY scripts/docker-entrypoint.sh /usr/local/bin/
+COPY --from=postgres_rpms /rpms/postgres.rpm /rpms/postgres-libs.rpm /rpms/postgres-server.rpm /rpms/postgres-contrib.rpm /tmp/
 
 ARG POSTGRESQL_ARCH=x86_64
 
-RUN curl -sSLf https://download.postgresql.org/pub/repos/yum/RPM-GPG-KEY-PGDG-${PG_MAJOR} -o /tmp/pg_repo.key && \
-    rpm --import /tmp/pg_repo.key && \
-    curl -sSLf https://download.postgresql.org/pub/repos/yum/reporpms/EL-8-${POSTGRESQL_ARCH}/pgdg-redhat-repo-latest.noarch.rpm -o /tmp/pg_repo.rpm && \
-    rpm -i /tmp/pg_repo.rpm && \
-    # Skip repo check for ARM as it's used only for development purposes and package has wrong signature.
-    if [[ "$POSTGRESQL_ARCH" == "aarch64" ]]; then sed -i 's/repo_gpgcheck = 1/repo_gpgcheck = 0/g' /etc/yum.repos.d/pgdg-redhat-all.repo; fi && \
-    microdnf upgrade -y && \
+RUN microdnf upgrade -y && \
+    # groupadd is in shadow-utils package that is not installed by default.
     microdnf install -y shadow-utils && \
     groupadd -g 70 postgres && \
     adduser postgres -u 70 -g 70 -d /var/lib/postgresql -s /bin/sh && \
     microdnf install -y \
-        ca-certificates libicu systemd-sysv glibc-locale-source glibc-langpack-en \
-        postgresql13-server && \
-    # The removal of /usr/share/zoneinfo from UBI minimal images is intentional.
-    # After building the image, the image is reduced in size as much as possible,
-    # and the /usr/share/zoneinfo directory is purged as it saves space
-    # in the final distribution of the image.
+        ca-certificates libicu systemd-sysv \
+        glibc-locale-source glibc-langpack-en \
+        perl-libs libxslt && \
```

Comment:
copied here from the central-db Dockerfile. Curious, are these really needed? @connorgorman

Reply:
I actually don't know. I think @c-du wrote that one and @janisz edited?

Reply:
https://github.com/stackrox/stackrox/pull/3880/files#r1029560377

Guess Scanner didn't need this before because of how postgres was installed